src/ideation/dto/create-ideation-vote.dto.ts

```diff
@@ -0,0 +1,9 @@
+import { ApiProperty } from "@nestjs/swagger";
+
+export class CreateIdeationVoteDto {
+    @ApiProperty()
+    votedBy: string;
+    userId: number;
```

Comment:
is this duplicated? votedBy is already the userId (uuid)

Reply:
I believe so, looking at the schema it looks like this is where the relation is.